src/server.js

```diff
@@ -19,6 +20,37 @@ function createServer(cluster) {
   });
 
   server.route({
+    method: 'GET',
+    path: '/healthcheck',
+    handler: (request, reply) => {
+      // Exposes an endpoint to report on server health
+
+      const authClient = new CloudFoundryAuthClient();
+
+      // Array of promises returned from methods we want included in the healthcheck
+      const checkPromises = [
```

Comment:
I set this up so we could "easily" add additional checks, though right now there is only one